#!/usr/bin/env python
PACKAGE = "osm_localization"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("deck_size", int_t,         0, "pcls accumulated to build aggregate",    7, 1,   15)
gen.add("period",    double_t,      0, "how often we make ground",               4, 1,   10)
gen.add("norm_threshold", double_t, 0, "normal filter",                       0.95, 0,   1)

gen.add("search_radius", double_t,  0, "how far to look for nearest neigh",   0.2,  0.05, 3)
gen.add("centroid_dist",   double_t,   0, "Distance deviation to deem edge",    0.5,  0,   3)  

exit(gen.generate(PACKAGE, "osm_localization", "osm_localization"))